src/Middleware/ConcurrencyLimiter/src/ConcurrencyLimiterExtensions.cs

```diff
@@ -15,6 +15,7 @@ public static class ConcurrencyLimiterExtensions
     /// </summary>
     /// <param name="app">The <see cref="IApplicationBuilder"/>.</param>
     /// <returns>The <see cref="IApplicationBuilder"/>.</returns>
+    [Obsolete()]
```

Comment:
Why is this message empty?

Reply:
It's a rebel. It has been crushed in one swift stroke.